web/components/new-contract-panel.tsx

```diff
@@ -132,6 +133,17 @@ export function NewContractPanel(props: {
     ? parseFloat(initialValueString)
     : undefined
 
+  useEffect(() => {
+    if (outcomeType === 'STONK') {
+      setInitialValueString('5000')
+      setMaxString('10000')
```

Comment:
Might want to just hard code these as constants for all stonks, instead of having them saved as variables. This approach is more brittle and might make it harder to migrate to other mechanisms in the future.

Reply:
Yeah, have the min and max for all stonks be the same.